docs/runbooks.md

```diff
@@ -0,0 +1,23 @@
+# OpenTelemetry Operator Runbooks
+
+See [`README.md`](../README.md) for more details about the OpenTelemetry Operator.
+
+## Manager Rules
+
+### [ReconcileErrors](#reconcileerrors)
+
+|||
+|-:|-|
+| Meaning | The OpenTelemetry Operator is unable to succeed in the reconciliaton step, probably because of a missconifgured OpenTelemetryCollector. |
+| Impact | No impact on already running deployments or new correct ones. |
+| Diagnosis | Check manager logs for reasons why this might happen. <br> <br> Example: `failed to create objects for <podName>: Deployment.apps "<deploymentName>" is invalid: spec.template.spec.containers[0].env[1].valueFrom: Invalid value: "": may not be specified when "value" is not empty`|
```

Comment:
should we include a real example of this? I believe the operator should be preventing most of these already, so having a clear example and how its solved may be useful. (we're also working on preventing these from happening with #2193)

Reply:
That is a real text from an alert we had. Do you mean i should add the OtelCol config that caused the alert?

Reply:
oh I hadn't seen this from the operator before so I assumed it was contrived, if it was legit then we can keep it here.

Reply:
I could remove it. Now that I think more of it i feel like it only adds confusion. It would only help someone with that exact problem if i added some more resolution steps for that particular error. The operator logs are descriptive enough to find the problem.